scss/_card.scss

```diff
@@ -70,8 +70,42 @@
 
 // >> Landing Grid
 .card-landing-grid {
-  &:hover {
-    background-color: $gray-200;
+  border-style: solid;
+  border-width: 2px 0 0 2px;
+  box-shadow: 3px 3px 0px 0px rgba($sky, .5);
```

Comment:
```suggestion
  box-shadow: 3px 3px 0 0 rgba($sky, .5);
```

Reply:
Oh Sorry Martin I didn't see this comment for some reason. I though it was wanting units instead of no units so I just pushed that up. I will correct to remove all the units from the 0s. :)